Source/ARTRestPresence.m

```diff
@@ -130,6 +161,8 @@ - (BOOL)get:(ARTPresenceQuery *)query callback:(ARTPaginatedPresenceCallback)cal
         };
     }
 
+    query.frozen = true;
```

Comment:
**suggestion:** Consider alternative to exceptions for indicating frozen state

Instead of using exceptions to indicate that a query is frozen, consider returning a boolean value or using an error parameter. This aligns better with Objective-C conventions and avoids using exceptions for control flow.